TUTORIAL.md

```diff
@@ -0,0 +1,57 @@
+# Guide to Solving Countdown to Midnight
+
+The first error message we are getting after running `rspec` is: 
+
+### `#countdown`
+
+```bash
+LOOPS
+  #countdown
+    outputs '<n> SECOND(S)!' string to STDOUT for each count (FAILED - 1)
+    returns HAPPY NEW YEAR! (FAILED - 2)
+  #countdown_with_sleep
+    should take at least 5 seconds to execute (FAILED - 3)
+
+Failures:
+
+  1) LOOPS #countdown outputs '<n> SECOND(S)!' string to STDOUT for each count
+     Failure/Error: expect { countdown(10) }.to output(countdown_output).to_stdout
+     ArgumentError:
+       wrong number of arguments (1 for 0)
+```
+
+`Wrong number of arguments (1 for 0)` means our countdown method expects from us that we pass in an argument.
```

Comment:
"means our countdown method expects one argument (the `1` part of the error) but we passed in 0 (the `0` part of the error)". 

Important to link what you say, to how you knew that